src/pages/projectSettings/tabs/GeneralTab/getFormSchema.tsx

```diff
@@ -342,7 +351,8 @@ export const getFormSchema = (
         repotrackerDisabled: {
           "ui:widget": widgets.RadioBoxWidget,
           "ui:description":
-            "Repotracker will be triggered from GitHub push events sent via webhook.",
+            "Repotracker will be triggered from GitHub push events sent via webhook. " +
```

Comment:
<img width="569" alt="image" src="https://user-images.githubusercontent.com/13104717/169374651-bd4ea0fc-afba-462e-b3c0-010b21212866.png">
With the description, the **Repotracker** subtitle now feels unnecessary to me.